applications/pelion_client/src/modules/net_ot.c

```diff
@@ -200,6 +206,25 @@ static bool event_handler(const struct event_header *eh)
 			module_set_state(MODULE_STATE_READY);
 			k_delayed_work_submit(&connecting_work, K_NO_WAIT);
 		}
+		return false;
+	}
+
+	if (IS_ENABLED(CONFIG_PELION_CLIENT_FACTORY_RESET_EVENTS) && is_factory_reset_event(eh)) {
```

Comment:
```
if (IS_ENABLED(CONFIG....) &&
    is_factory_reset_event(....)) {
}
```

Reply:
Well - it fits 100 characters limit - I prefer to have it in one line, but we can split it as well.

Reply:
I mention it only because I find it easier to read if it's split. Since it's style if you prefer to leave, left it will be.